healthcare/api-client/v1/hl7v2/hl7v2_stores_test.py

```diff
@@ -0,0 +1,223 @@
+    def create():
+        try:
+            hl7v2_stores.create_hl7v2_store(
+                project_id, cloud_region, dataset_id, hl7v2_store_id)
+        except HttpError as err:
+            # We ignore 409 conflict here, because we know it's most
+            # likely the first request failed on the client side, but
+            # the creation suceeded on the server side.
+            if err.resp.status == 409:
+                print(
+                    'Got exception {} while creating HL7v2 store'.format(
+                        err.resp.status))
+            else:
+                raise
+    create()
+
+    yield
+
+    # Clean up
+    @retry(
+        wait_exponential_multiplier=1000,
+        wait_exponential_max=10000,
+        stop_max_attempt_number=10,
+        retry_on_exception=retry_if_server_exception)
+    def clean_up():
+        try:
+            hl7v2_stores.delete_hl7v2_store(
+                project_id, cloud_region, dataset_id, hl7v2_store_id)
+        except HttpError as err:
+            # The API returns 403 when the HL7v2 store doesn't exist.
+            if err.resp.status == 404 or err.resp.status == 403:
+                print(
+                    'Got exception {} while deleting HL7v2 store'.format(
+                        err.resp.status))
+            else:
+                raise
+
+    clean_up()
+
+
+@pytest.fixture(scope="module")
+def crud_hl7v2_store_id():
+    yield hl7v2_store_id
+
+    # Clean up
+    @retry(
+        wait_exponential_multiplier=1000,
+        wait_exponential_max=10000,
+        stop_max_attempt_number=10,
+        retry_on_exception=retry_if_server_exception)
+    def clean_up():
+        try:
+            hl7v2_stores.delete_hl7v2_store(
+                project_id, cloud_region, dataset_id, hl7v2_store_id)
+        except HttpError as err:
+            # The API returns 403 when the HL7v2 store doesn't exist.
+            if err.resp.status == 404 or err.resp.status == 403:
+                print(
+                    'Got exception {} while deleting HL7v2 store'.format(
+                        err.resp.status))
+            else:
+                raise
+
+    clean_up()
+
+
+def test_CRUD_hl7v2_store(test_dataset, crud_hl7v2_store_id, capsys):
+    hl7v2_stores.create_hl7v2_store(
+        project_id,
+        cloud_region,
+        dataset_id,
+        hl7v2_store_id)
+
+    hl7v2_stores.get_hl7v2_store(
+        project_id,
+        cloud_region,
+        dataset_id,
+        hl7v2_store_id)
+
+    hl7v2_stores.list_hl7v2_stores(
+        project_id,
+        cloud_region,
+        dataset_id)
+
+    hl7v2_stores.delete_hl7v2_store(
+        project_id,
+        cloud_region,
+        dataset_id,
+        hl7v2_store_id)
+
+    out, _ = capsys.readouterr()
+
+    # Check that create/get/list/delete worked
+    assert 'Created HL7v2 store' in out
+    assert 'Name' in out
+    assert 'hl7V2Stores' in out
+    assert 'Deleted HL7v2 store' in out
```

Comment:
These asserts should probably happen after each function is run to avoid one function from accidentally testing positive for all 3